packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -111,6 +111,17 @@ TextSpan truncateTextSpan(TextSpan span, int length) {
   return truncateHelper(span);
 }
 
+/// Returns a [double] representing the width in pixels of the [span].
```

Comment:
nit on doc comment writing. Try to avoid  saying things in doc comments that are obvious from the method name and signature.
For example, this comment can be written as
```dart
/// Returns the width in pixels of the [span].
```
without a reduction in clarity.

Reply:
Makes sense, updated the doc comment!